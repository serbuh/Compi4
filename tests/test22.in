// Blend of control structures

C x = 100 C;
F y = 100 F;
K z = 100 C;

if(x==z) {
  print "In first if - Good";
  if(z==x) {
    print "In second if - Good";
    int i = 5;
    while(i > 0) {
      y = y + 10 F;
      i = i - 1;
      while(x==x) {
        break;
      }
    }
  }
}

print y;
