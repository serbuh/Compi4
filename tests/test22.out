In first if - Good
In second if - Good
150
